go/vt/vttablet/endtoend/framework/server.go

```diff
@@ -111,6 +113,9 @@ func StartServer(connParams, connAppDebugParams mysql.ConnParams, dbName string)
 	config.TwoPCCoordinatorAddress = "fake"
 	config.HotRowProtection.Mode = tabletenv.Enable
 	config.TrackSchemaVersions = true
+	config.GracePeriods.ShutdownSeconds = 2
+	gotBytes, _ := yaml2.Marshal(config)
+	log.Infof("Config:\n%s", gotBytes)
```

Comment:
It's good to publish this on startup. VTTablet does the same thing. It's useful for knowing what the default values are for each flag. But it doesn't spam the output because it shows up only if you `-alsologtostderr`.